graph.js

```diff
@@ -611,6 +611,21 @@ class Graph {
         this.update();
     }
 
+    formatAttributes(attributes) {
+        const suffix = "Attributes";
+        let attrMap = new Map();
+        attributes.forEach(attr => {
+            const attrType = attr.type + suffix;
+            let attrObj = {};
+            if (attrMap.has(attrType)) {
+                attrObj = attrMap.get(attrType);
+            }
+            attrObj[attr.key] = attr.value;
```

Comment:
this makes all the output as just string. Would be better if we can do like numeric attributes and boolean attributes to not be a string. Example: {"someBoolean": true} instead of {"someBoolean": "true"}

Reply:
What is the edge id? In the code there is this function
```
    edgeId(d) {
        return String(d.source.id) + "+" + String(d.target.id);
    }
```
But is it actually supposed to be some unique number rather than a concat of source and target id?